style1.css

```
@@ -0,0 +1,134 @@
+
+@font-face {
+    font-family: Roboto;
+    src: url('https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap');
+}
+body {
+    margin: 20px;
+}
+
+.search_bar_1 {
+   position: relative;
+   margin-top: 20px;
+   width: 100%;
+   font-weight: 300;
+   background: #FFFFFF;
+   border: 1px solid #E1E7ED;
+   box-sizing: border-box;
+   box-shadow: 0px 1px 8px rgba(61, 78, 97, 0.1);
+   border-radius: 4px;
+   height: 70px;
+   
+}
+
+.search_bar_1:focus {
+    position: relative;
+    margin-top: 20px;
+    width: 100%;
+    font-weight: 300;
+    background: linear-gradient(180deg, #FFFFFF 0%, #F6F6F7 100%);
+    border: 1px solid #E1E7ED;
+    box-sizing: border-box;
+    border-radius: 3px;
+    height: 70px;
+}
+
+.search_bar_1:hover {
+    position: relative;
+    margin-top: 20px;
+    width: 100%;
+    font-weight: 300;
+    background: #FFFFFF;
+    border: 1px solid #E1E7ED;
+    box-sizing: border-box;
+    box-shadow: 0px 3px 8px rgba(61, 78, 97, 0.2);
+    border-radius: 4px;
+    height: 70px;
+    
+ }
+
+
+ .box {
+    position: relative;
+    margin-top: 20px;
+    width: 100%;
+    font-weight: 300;
+    background: #FFFFFF;
+    border: 1px solid #E1E7ED;
+    box-sizing: border-box;
+    box-shadow: 0px 1px 8px rgba(61, 78, 97, 0.1);
+    border-radius: 4px;
+    height: 50px;
+    
+ }
+ 
+ .box:focus {
+     position: relative;
+     margin-top: 20px;
+     width: 100%;
+     font-weight: 300;
+     background: linear-gradient(180deg, #FFFFFF 0%, #F6F6F7 100%);
+     border: 1px solid #E1E7ED;
+     box-sizing: border-box;
+     border-radius: 3px;
+     height: 50px;
+ }
+ 
+ .box:hover {
+     position: relative;
+     margin-top: 20px;
+     width: 100%;
+     font-weight: 300;
+     background: #FFFFFF;
+     border: 1px solid #E1E7ED;
+     box-sizing: border-box;
+     box-shadow: 0px 3px 8px rgba(61, 78, 97, 0.2);
+     border-radius: 4px;
+     height: 50px;
+     
+  }
+
+.photo {
+    position: relative;
+    width: 19px;
+    height: 19px;
+    left: 26px;
+    top: 25px;
+}
+
+.text {
+
+position: relative;
+top: 20px;
+left: 50px;
+
+
+font-size: 16px;
+line-height: 22px;
+border: none;
+}
```

```suggestion
.text {
  position: relative;
  top: 20px;
  left: 50px;
  font-size: 16px;
  line-height: 22px;
  border: none;
}
```
Форматування. Не варто додавати так багато пустих рядків